tests/functional_tests/missing_files.bats

```diff
@@ -33,8 +48,16 @@ teardown() {
 
 #Note : The output of this test will differ if any changes are made to docs/examples/unit_tests/execute.sh
 @test "run no test info found" {
-  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/extract_run.sh
-  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_1/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_2/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_3/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_4/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_5/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_1/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_2/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_3/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_4/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_5/extract_run.sh
   node submit.js -i 2015A7PS006G -l lab1 --lang=java --host='localhost:9000' > \
```

Comment:
changes required here to exercise all the five execution nodes. As you mentioned, we need to have tests for cases illustrated in issue #206.